composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/domain/model/Event.kt

```diff
@@ -3,6 +3,7 @@ package sp.bvantur.tasky.event.domain.model
 import kotlinx.datetime.LocalDateTime
 
 data class Event(
+    val eventId: String?,
```

Comment:
Considering that the ID is set locally, and you don’t rely on a remote server for ID generation, won’t it make more sense to have this be non-nullable?